frontend/src/AppRoutes/index.tsx

```diff
@@ -62,10 +62,12 @@ function App(): JSX.Element {
 			allFlags.find((flag) => flag.name === FeatureKeys.ONBOARDING)?.active ||
 			false;
 
-		const isChatSupportEnabled =
+		const isChatSupportFFEnabled =
 			allFlags.find((flag) => flag.name === FeatureKeys.CHAT_SUPPORT)?.active ||
 			false;
 
+		const isChatSupportEnabled = isChatSupportFFEnabled;
```

Comment:
NIT: Didn't get why this is done.

Reply:
Waiting for the API to get the required data.

Reply:
Updated.